src/app-store/definitions/uptime-kuma/uptime-kuma.lisp

```diff
@@ -0,0 +1,10 @@
+(define-app
+    (version "1.23.15")
+    (ports 3001)
+        (containers
+            (container
+                (name "uptime-kuma")
+                (image "docker.io/louislam/uptime-kuma:1")
```

Comment:
use more granular version pls

Reply:
like this: https://hub.docker.com/layers/louislam/uptime-kuma/1.23.15/images/sha256-8eea954901871d3e33f2dbddf9d6be07d362b0fc01c4ec613f390d4bd3d99cf2?context=explore